index.js

```diff
@@ -3,7 +3,10 @@
  * @param {object} assignments - An object with assignments organized by category.
  * @returns {string[]} An array of strings, which represent all of the keys in the object.
  */
-function getAssignmentTypes(assignments) {}
+function getAssignmentTypes(assignments) {
+  let keys = Object.keys(assignments)
+  return keys;
```

Comment:
You don't need this intermediate variable. Try returning the return value of `Object.keys` outright, though you're welcome to prefer the intermediate variable anyway!